include/cute/atom/copy_traits_xe.hpp

```diff
@@ -195,17 +265,21 @@ struct XE_2D_LD_Unpack {
   prefetch(Copy_Atom<Traits_LD_t, CA_Args...> const &atom,
            Tensor<TS, SLayout> const &src) {
     static_assert(detail::has_prefetch<CopyOp>);
+    //TODO(Codeplay) add asserts on size
 
     using dtype = typename Copy_Atom<Traits_LD_t, CA_Args...>::ValType;
 
     dtype *base_addr = (dtype *)atom.base_ptr;
 
     auto [m, n, l] = src.data().coord_;
+    
+    int x = is_need_reversed ? m : n;
+    int y = is_need_reversed ? n : m;
 
     CopyOp::PREFETCH::copy((void *)(base_addr + l * atom.stride_l),
                            atom.width * sizeof(dtype), atom.height,
                            atom.pitch * sizeof(dtype),
-                           intel::coord_t{(int)n, (int)m});
+                           intel::coord_t{x, y});
```

Comment:
So this is not neither

Reply:
reverted

Reply:
Actually I was wrong to revert this. This switch is not about the copy operations (which we only have LD_N for prefetch), but about the matrix layout. So this is required for either col-major matrix A or row-major matrix B.

Reverting the revert.

Reply:
Yes I saw that you actually fix the coordinate layout here : https://github.com/codeplaysoftware/cutlass-fork/pull/249/files#diff-8655552320737c72499d330805f4033a4f99177be86e4262c8fd9c817f15762eR187
So that the coordinate for prefetch can now be (m,k)/(n,k) similar to load function. Hence this is needed.